safekeeper/src/send_wal.rs

```diff
@@ -262,8 +275,9 @@ impl WalSendersShared {
 
     /// Update aggregated hot standy feedback. We just take min of valid xmins
```

Comment:
Function now updates both hot standby and normal reply feedback, let's update comment.

Reply:
Done